app.go

```diff
@@ -11,6 +11,15 @@ import (
 	"time"
 )
 
+var (
+	errNonFuncAction = NewExitError("ERROR invalid Action type.  "+
```

Comment:
I like the addition of the hyperlink, but I still think it'd be nice to include something like: `This is an error in the application, please contact the distributor of this application if this is not you` just in case I, as a non-programmer, install a program using `codegangsta/cli` and receive this error message.

Harkening back to `Please contact your system administrator` which was always amusing when that was indeed you.